### ISSUE TITLE
Controller D-Pad echo events stopped working after version 3.2.3

### ISSUE THREAD

**Issue by mrkdji** [godotengine/godot#56437]
### Godot version

3.4.2

### System information

Windows 10

### Issue description

I am aware that the current _Controllers, gamepads, and joysticks_ section of the docs specify:

> **"Echo" events**
> Unlike keyboard input, holding down a controller button such as a D-pad direction will not generate repeated input events at fixed intervals (also known as "echo" events). This is because the operating system never sends "echo" events for controller input in the first place.
> 
> If you want controller buttons to send echo events, you will have to generate InputEvent objects by code and parse them using Input.parse_input_event() at regular intervals. This can be accomplished with the help of a Timer node.

Nevertheless I noticed that D-pad echo events work as expected in version 3.2.3 and stopped doing so in 3.2.4 and in every other Godot version >= 3.2.4. 

Was this feature purposefully removed? If so why?

*Note:* I'm using the Switch Pro Controller.

### Steps to reproduce

Add a Sprite to a scene and attach some code to make it move (such as the one in the Minimal reproduction project).
Open the project with Godot 3.2.3 and try to move the sprite using the D-pad of an external controller (it should move), 
then try the same with any version >= 3.2.4-rc5 ( it shouldn't move, I also tried with versions 3.3, 3.3.4, 3.4.2 )

### Minimal reproduction project

```
extends Sprite

const SPEED = 100;
var input : Vector2 = Vector2.ZERO;

func _physics_process(delta):
	
	input.x = Input.get_action_strength("ui_right") - Input.get_action_strength("ui_left");
	input.y = Input.get_action_strength("ui_down") - Input.get_action_strength("ui_up");
	input = input.normalized();
	
	global_position += input * SPEED * delta;
	pass;
```

**Comment by Calinou** [comment]
Related to https://github.com/godotengine/godot/issues/54959.

**Comment by mrkdji** [comment]
> Related to #54959 (UI focus change is triggered too often when using gamepad analog stick to navigate menus)

I tested that issue with the Nintendo Pro Controller and I have no problem changing focus with the analog stick both in 3.2.3 and 3.4.2.
Don't know if it can be of any help but I noticed that in 3.4.2 moving the analog stick calls _gui_input(event) / event.is_action_pressed continuously (the expected behaviour I think?) while in 3.2.3 it calls it 3-4 times on each movement.

**Comment by mrkdji** [comment]
I just noticed that the D-pad input works also in Godot versions > 3.2.3 but only if the action is not mapped to a joystick axis

**Comment by madmiraal** [comment]
> I just noticed that the D-pad input works also in Godot versions > 3.2.3 but only if the action is not mapped to a joystick axis as well

This is a duplicate of #45628, which is fixed with #47599 and it's 3.x version #48343.

**Comment by akien-mga** [comment]
Duplicate of #45628 as described above.
